README.md

```diff
@@ -4,14 +4,20 @@
 
 ---
 #### Table of Contents
-* [Want to contribute?](#want-to-contribute)
-    * [Requirements](#requirements)
-    * [First-Time Setup](#first-time-setup)
-        * [DB Setup](#db-setup-with-ms-sql)
-        * [Slack Setup](#slack-setup)
-    * [Local Configuration Files](#local-configuration-files)
-* [Running & Debugging (VS Code)](#running--debugging-vs-code)
-* [Helpful Links](#helpful-links)
+- [Egg Party 🥚🎉](#egg-party-%f0%9f%a5%9a%f0%9f%8e%89)
+    - [Egg party is a Slack App for thanking teammates, raising chickens, and having a rowdy ol' time](#egg-party-is-a-slack-app-for-thanking-teammates-raising-chickens-and-having-a-rowdy-ol-time)
+      - [Table of Contents](#table-of-contents)
+  - [* Helpful Links](#helpful-links)
```

Comment:
did you mean to bold "Helpful Links"? I like the new bolded style of the main content items!

Reply:
Actually Helpful Links is already at the bottom of the ToC. Is this one maybe an accident?

Reply:
Hey @cakekindel ... actually, the only items I changed in the README were the addition of the DB Troubleshooting Notes.  Everything else I left as is...